src/main/java/com/sprint/mission/discodeit/service/domain/ReadStatusService.java

```diff
@@ -0,0 +1,104 @@
+package com.sprint.mission.discodeit.service.domain;
+
+import com.sprint.mission.discodeit.entity.ReadStatus;
+import com.sprint.mission.discodeit.dto.ReadStatusDto;
+import com.sprint.mission.discodeit.entity.Channel;
+import com.sprint.mission.discodeit.entity.User;
+import com.sprint.mission.discodeit.repository.ChannelRepository;
+import com.sprint.mission.discodeit.repository.UserRepository;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+
+import java.time.Instant;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+
+@Service
+@RequiredArgsConstructor
+public class ReadStatusService{
+    //사용자 별 마지막으로 확인된 접속 시간을 표현하는 도메인 모델입니다. 사용자의 온라인 상태를 확인하기 위해 활용합니다.
+    private final com.sprint.mission.discodeit.repository.ReadStatusRepository readStatusRepository;
```

Comment:
```suggestion
    private final ReadStatusRepository readStatusRepository;
```
패키지 경로까지 잘못 들어가있네요! 확인 바랍니당